test/cassettes/test_formula_template.rb

```diff
@@ -3,12 +3,15 @@
 
 # This file was generated by Homebrew Releaser. DO NOT EDIT.
 class HomebrewReleaser < Formula
-  desc "Tool to release scripts, binaries, and executables to github"
```

Comment:
Once the description is returned to how it shows up in the test, this field should not be altered in this PR. (see notes below)